cpp/src/linear_programming/cuopt_c.cpp

```diff
@@ -74,6 +76,29 @@ cuopt_int_t cuOptGetVersion(cuopt_int_t* version_major,
   return CUOPT_SUCCESS;
 }
 
+bool extraTimestamps() {
+    const char* envValue = std::getenv("CUOPT_EXTRA_TIMESTAMPS");
```

Comment:
If we can get it once and store and reuse this, we don't have to request env name every-time, but this also depends on how many times this function is going to be called. 

And would this be helpful if we move it out and make it common for routing and linear programming ?